packages/plugins/rpc-manager/src/lib/eth_sendTransaction.ts

```diff
@@ -73,9 +78,13 @@ export default class EthSendTransaction extends RpcModifier {
           if (err) {
             return callback(err, null);
           }
+          params.originalRequest = cloneDeep(params.request);
           params.request.method = blockchainConstants.transactionMethods.eth_sendRawTransaction;
           params.request.params = [newPayload];
-          callback(err, params);
+          // allow for any mods to eth_sendRawTransaction
+          this.plugins.runActionsForEvent('blockchain:proxy:request', params, (error, result) => {
```

Comment:
Could probably just be `runActionsForEvent('blockchain:proxy:request', callback)`

Reply:
Good catch, hangover from debugging. Updated.